mobile_verifier/tests/integrations/hex_boosting.rs

```diff
@@ -990,6 +1007,163 @@ async fn test_poc_with_cbrs_and_multi_coverage_boosted_hexes(pool: PgPool) -> an
     Ok(())
 }
 
+#[sqlx::test]
+async fn test_poc_boosted_hex_stack_multiplier(pool: PgPool) -> anyhow::Result<()> {
+    // This test seeds the database with 2 identically performing radios on
+    // neighboring hexes. Both are eligible for boosted rewards.
+    //
+    // In the first pass, no hexes are boosted, to ensure the radios are equal.
+    // In the second pass, 1 radio is boosted with 2 boosts.
+    //  - legacy boost,   BoostType::All
+    //  - specific boost, BoostType::CbrsIndoor
+    // Rewards are checked to ensure the 2 boosts accumulate correctly.
```

Comment:
can we get a test  to include boosted hex entries that cover each of the device types

Reply:
What's the goal?

We can add a boosted hex of each type, to make sure only `::All` boosted hexes are added to individual radios types.

If you're looking for every radio type getting rewarded with different boosts applied... The only way I can think to make any meaningful assertions against those rewards would be to:
1. hardcode the reward values, which we're trying to stay/get away from
2. do the reward math in the test, which would need to change as often as the actual reward calculations.

There's also an option of doing a second pass for each type of radio. But if all radio types are involved in each pass I think that would put us back in the situation of, the only meaningful assertions we could make are against concrete numbers, or doing complex math in the test.

We could also flesh out the unit test `boosted_hex_info::tests::boosted_hexes_accumulate_multipliers` to go through all device types.

Thoughts?

Reply:
The goal would be to ensure that when a hex is boosted for each specific device type only that type gets boosted rewards.  If you want to avoid hardcoding reward amounts you can assert on relative values, you could setup two hexes each with a cbrs and a wifi radio.  Hex 1 is boosted for CBRS with a multiplier of 5x, hex 2 has no boosts.  The cbrs radio in hex1 should have 5x the reward amount of the cbrs radio in hex2.  The wifi radios in both hexes should have the same reward

Reply:
Gotcha, that sounds like a good test. 
Added here https://github.com/helium/oracles/pull/798/commits/dc181f87ff99a5bd6c88699690864a08b5fdaa9e